features/steps/thens/existance.py

```diff
@@ -22,3 +22,10 @@ def step_impl(context, inst, constraint, num, entity):
 
         if not op(len(instances_in_model), num):
             yield StepResult(expected=num, observed=len(instances_in_model))
+
+
+@validate_step('There must be one representation item')
+def step_impl(context, inst):
+
+    count = len(inst.Representation.Items)
+    yield StepResult(expected=142, observed=count)
```

Comment:
```suggestion
    yield StepResult(expected=1, observed=count)
```

But as said above, we should probably find a way to make this more generic. I'm also a bit confused actually on why you do `inst.Representation.Items` here. I would have imagined that by the time you would evaluate this statement, `inst` would already be the value of `Items` due to `Its attribute Items`.

Reply:
Correct, at this point in the process `inst` is the Items attribute.  It appears that inst will always be a list of length 1 but the list object will contain one or more `Items` in a tuple.  Therefore I have revised the step implementation to check both the length of the list and and the length of the first list item.